queries/scss/highlights.scm

```diff
@@ -6,6 +6,10 @@
   "@while"
   "@each"
   "@include"
+  "@for"
+  "from"
+  "through"
+  "in"
 ] @keyword
```

Comment:
we could take this chance to make all loop related keywords `@repeat`

Reply:
Done